auto_tests/conference_test.c

```diff
@@ -268,43 +250,43 @@ static void run_conference_tests(Tox **toxes, State *state)
 
         for (uint32_t i = 0; i < NUM_GROUP_TOX; ++i) {
             char name[NAMELEN + 1];
-            snprintf(name, NAMELEN + 1, NEW_NAME_FORMAT_STR, state[i].index);
-            tox_self_set_name(toxes[i], (const uint8_t *)name, NAMELEN, nullptr);
+            snprintf(name, NAMELEN + 1, NEW_NAME_FORMAT_STR, autotoxes[i].index);
+            tox_self_set_name(autotoxes[i].tox, (const uint8_t *)name, NAMELEN, nullptr);
         }
     }
 
     for (uint16_t i = 0; i < NUM_GROUP_TOX; ++i) {
-        const uint32_t num_frozen = tox_conference_offline_peer_count(toxes[i], 0, nullptr);
+        const uint32_t num_frozen = tox_conference_offline_peer_count(autotoxes[i].tox, 0, nullptr);
         ck_assert_msg(num_frozen <= max_frozen,
                       "tox #%u has too many offline peers: %u\n",
```

Comment:
c-toxcore/auto_tests/conference_test.c:260: failed `num_frozen <= max_frozen': tox #3 has too many offline peers: 15

This started happening for me in this PR.